js/script.js

```diff
@@ -205,11 +233,15 @@ function popwork(id) {
   </ul>
 </div>
   <div class="side-pop">
-    <img class="tonic" alt= " ${cards[id].alt}" src="${cards[id].image}">
+    <img class="tonic" id ="mob" alt= " ${cards[id].alt}" src="${cards[id].image}">
+    <img class="tonic hide" alt= " ${cards[id].alt}" src="${cards[id].imaged}">
   </div>
+  <div class ="work-lang">
     <div class="work">
-    <p>${cards[id].description}</p>
+    <p id ="mob">${cards[id].description}</p>
+    <p class ="hide">${cards[id].descriptiond}</p>
   </div>
+  <div class ="lang-see">
   <ul class="langw">
```

Comment:
- [ ] This is an intelligent attempt :+1:  however, to give users a more compelling experience, I think it will be nice to look into some issues. Let's see how your page looks like when compared with Figma's:

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/225369659-a54ed5af-7e2e-471f-9a62-e95c6b8c7103.png)|![image](https://user-images.githubusercontent.com/99715640/225369751-5eaf2adf-a273-4b78-a3cc-92038a6106ae.png)|
|Your Desktop View| Figma's Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/225369895-dafbec80-8b37-4f5e-90cb-522e69bb6a63.png)|![image](https://user-images.githubusercontent.com/99715640/225369823-2e621fa8-532f-4fbd-8a55-f1de3862ed1b.png)|

#### Here are a few points to note:
- Your pop-up is not well aligned at the center of the page whether on mobile or desktop
- Other parts of the page are still visible after the user has clicked on `See Project` button, which is not best for user experience. Kindly apply some blurring effects on the background as contained in the Figma design
- Your button texts seem not to be well centralized which isn't good for your aesthetics. Kindly adjust that.
- Also, I think you have an arbitrary height on the pop-up, that makes it too long than necessary :+1: